modules/src/xkcd.py

```
@@ -32,9 +33,10 @@ def process(input, entities=None):
         template = GenericTemplate()
         template.add_element(title=title, item_url=item_url, image_url=image_url, subtitle=subtitle,
                              buttons=buttons.get_buttons())
-
+        message = template.get_message()
+        message = add_quick_reply(message, 'Check out another!', modules.generate_postback('xkcd'))
```

Ignore this error for now. I've set up max line length to 120.